transport/src/main/java/io/netty/channel/DefaultChannelPipeline.java

```diff
@@ -159,10 +161,7 @@ public ChannelPipeline addBefore(String baseName, String name, ChannelHandler ha
     @Override
     public synchronized ChannelPipeline addBefore(
             EventExecutorGroup group, String baseName, String name, ChannelHandler handler) {
-        AbstractChannelHandlerContext ctx = getContextOrDie(baseName);
-        name = filterName(name, handler);
-        addBefore0(ctx, new DefaultChannelHandlerContext(this, findInvoker(group), name, handler));
-        return this;
+        return addBefore(findInvoker(group), baseName, name, handler);
```

Comment:
Is this change necessary for this PR? If not can we revert this change and submit in a new PR?